test/spec/modules/dailymotionBidAdapter_spec.js

```diff
@@ -1834,6 +2093,181 @@ describe('dailymotionBidAdapterTests', () => {
     });
   });
 
+  describe('validates buildRequests for video metadata iabcat1 and iabcat2', () => {
+    let bidRequestData = [{
+      auctionId: 'b06c5141-fe8f-4cdf-9d7d-54415490a917',
+      bidId: 123456,
+      adUnitCode: 'preroll',
+      mediaTypes: {
+        video: {
+          api: [2, 7],
+          startdelay: 0,
+        },
+      },
+      sizes: [[1920, 1080]],
+      params: {
+        apiKey: 'test_api_key',
+        video: {
+          iabcat1: ['video-params-iabcat1'],
+          iabcat2: ['video-params-iabcat2'],
+        },
+      },
+    }];
+
+    let bidderRequestData = {
+      timeout: 4242,
+      refererInfo: {
+        page: 'https://publisher.com',
+      },
+      ortb2: {
+        site: {
+          content: {
+            data: [
+              {
+                name: 'dataprovider.com',
+                ext: { segtax: 4 },
+                segment: [{ id: '1' }],
+              },
+              {
+                name: 'dataprovider.com',
+                ext: { segtax: 5 },
+                segment: [{ id: '6' }],
+              },
+              {
+                name: 'dataprovider.com',
+                ext: { segtax: 5 },
+                segment: [{ id: '17' }, { id: '20' }],
+              },
+            ]
+          },
+        }
+      }
+    };
+
+    config.setConfig({
+      userSync: {
+        syncEnabled: true,
+        filterSettings: {
+          image: {
+            bidders: ['dailymotion'],
+            filter: 'include'
+          },
+          iframe: {
+            bidders: ['dailymotion'],
+            filter: 'exclude',
+          },
+        }
+      }
+    });
```

Comment:
this should be in a setup block (`before` or `beforeEach`) and tore down with `config.resetConfig()`.

As it is here (outside of any test case) it's executed before any test, so unless you run your suite in isolation it will interfere with tests that do not expect it.